druid-shell/src/platform/gtk/window.rs

```diff
@@ -713,28 +713,26 @@ fn make_gdk_cursor(cursor: &Cursor, gdk_window: &gdk::Window) -> Option<gdk::Cur
     )
 }
 
-fn get_mouse_button(button: u32) -> MouseButton {
+fn get_mouse_button(button: u32) -> Option<MouseButton> {
     match button {
-        1 => MouseButton::Left,
-        2 => MouseButton::Middle,
-        3 => MouseButton::Right,
-        4 => MouseButton::X1,
-        5 => MouseButton::X2,
-        _ => MouseButton::Left,
+        0 => None,
+        1 => Some(MouseButton::Left),
+        2 => Some(MouseButton::Middle),
+        3 => Some(MouseButton::Right),
+        4 => Some(MouseButton::X1),
+        5 => Some(MouseButton::X2),
+        _ => Some(MouseButton::Unknown),
```

Comment:
I might consider making this `Unknown(u32)`? it doesn't matter too much either way, but if someone wants that info we might as well provide it?

But only if it's not a super big hassle..